Gopkg.toml

```diff
@@ -74,21 +74,21 @@ required = [
   name = "github.com/prometheus/client_golang"
   version = "0.9.3"
 
-[[constraint]]
+[[override]]
```

Comment:
This is rude, but seems to be the only way to work out the circle with e2e-libraries.